src/vaadin-grid-sorter.html

```diff
@@ -128,11 +129,13 @@
               value: null
             },
 
+            /** @private */
```

Comment:
Should probably be `@protected` since it is being set from `SortMixin`.

Reply:
Done.